dist/index.js

```diff
@@ -1988,6 +1988,7 @@ async function run() {
     const githubToken = core.getInput("GITHUB_TOKEN", { required: true });
     const pullRequestTitle = core.getInput("PULL_REQUEST_TITLE");
     const pullRequestBody = core.getInput("PULL_REQUEST_BODY");
+    const pullRequestIsDraft = core.getInput("PULL_REQUEST_IS_DRAFT");
```

Comment:
I would rather accept a boolean for the env variable. Considering that `core.getInput` returns a string, we could convert that in the code here.

```suggestion
    const pullRequestIsDraft = core.getInput("PULL_REQUEST_IS_DRAFT").toLowerCase() === "true";
```